src/Microsoft.AspNet.Mvc.Core/Internal/ResponseContentTypeHelper.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using Microsoft.AspNet.Mvc.Core;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.AspNet.Mvc.Internal
+{
+    public static class ResponseContentTypeHelper
+    {
+        public static void ResolveContentTypeAndEncoding(
+            MediaTypeHeaderValue actionResultContentType,
+            string httpResponseContentType,
+            MediaTypeHeaderValue defaultContentType,
+            out string resolvedResponseContentType,
+            out Encoding resolvedContentTypeEncoding)
```

Comment:
Suggest removing assumptions and redundancies from these names. `actionResultContentType` says something about the caller, not this code. Here it's more like `contentTypeOverride`. And the word "Response" doesn't add much in the `ResponseContentTypeHelper` class.